storage/innobase/xtrabackup/src/backup_mysql.cc

```diff
@@ -171,6 +171,17 @@ MYSQL *xb_mysql_connect() {
 
   xb_mysql_query(connection, "SET SESSION wait_timeout=2147483", false, true);
 
+  if (xb_mysql_numrows(connection, "SELECT @@wsrep_sync_wait", false) > 0) {
```

Comment:
@rahulmalik87 can we use have_galera_enabled here instead of doing a new query?

Reply:
fixed.